src/AllArticles.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+const style = {
+  padding: '20px 35px',
+  width: '78%',
+  margin: '10px auto',
+  borderRadius: '3px',
+  fontSize: '1.1em'
+}
+
+const padding = {
+  paddingBottom: '140px'
+}
```

Comment:
What's the purpose of using classes and applying styles directly? It seems to me that you should do one or the other.